spikeinterface/toolkit/postprocessing/tests/test_correlograms.py

```diff
@@ -19,5 +25,21 @@ def test_compute_correlograms():
     correlograms, bins = compute_correlograms(sorting2)
```

Comment:
You should loop over methods here.
And maybe the best would to check consistency across methods.

Reply:
Yup this is on my TODO list, I'll see when I have the time to do this.